processor/src/test/java/org/mapstruct/ap/test/source/constants/ConstantsMapper.java

```diff
@@ -29,11 +29,15 @@
  * @author Sjaak Derksen
  */
 @Mapper
-public interface NumericMapper {
+public interface ConstantsMapper {
 
-    NumericMapper INSTANCE = Mappers.getMapper( NumericMapper.class );
+    ConstantsMapper INSTANCE = Mappers.getMapper( ConstantsMapper.class );
 
     @Mappings({
+        @Mapping(target = "booleanValue", constant = "true"),
+        @Mapping(target = "booleanBoxed", constant = "false"),
+        @Mapping(target = "charValue", constant = "'b'"),
```

Comment:
well, I made it this way because, you can't confuse it with a string constant and its in line with integer/float assignments. After all, you can't do `setSomething( b )` or `something = b` as you can `setSomething( 5 )'`

Reply:
Makes sense